files/amazon-ssm-agent.gpg

```diff
@@ -0,0 +1,62 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
Q. Would it be easier for us to maintain only one gpg file in the [amazon-ecs-agent](https://github.com/aws/amazon-ecs-agent/blob/master/scripts/amazon-ssm-agent.gpg) repo? 

The pro I can think of is, only one file needs to be updated whenever SSM team rotated the key.
The con I can think of is, this will couple ECS Agent and ECS optimized AMI release together.

Reply:
I don't think we should. Open source the AMI repo helps people build AMI themselves and coupling these two definitely limits the flexibility.

Reply:
Discussed offline. We will continue maintaining both gpg files in ami and agent repos until the new mechanism from SSM Agent team is in place. Thanks!

https://docs.aws.amazon.com/systems-manager/latest/userguide/verify-agent-signature.html

Reply:
I agree with @Realmonia , having the 2 repos independent and flexible can be better here, as the 2 SSM agents in the different repos can be different, and the agent in amazon-ecs-ami repo is fixed by version. There is less need to introduce dependency this way.